commands/playwright-keplr.js

```diff
@@ -347,6 +358,13 @@ module.exports = {
     return extensionsData;
   },
 
+  async switchToKeplrRegistrationWindow() {
```

Comment:
nit: rename to `openKeplrRegistrationWindow` since we're opening a new window explicitly and not switching to an already opened one